src/uwtools/fs.py

```diff
@@ -54,8 +54,7 @@ def __init__(
                 **yaml_config.data,
             }
         )
-        self._config = yaml_config.data
-        self._set_config_block()
+        self._config, _ = walk_key_path(yaml_config.data, key_path or [])
```

Comment:
Reuse the existing `_walk_key_path()`, and delete the now-obsolete `_set_config_block()`, below.